src/core/Typing.ml

```diff
@@ -367,7 +367,7 @@ and check_constr cx dlbl param_specs constr params tms =
     | (lbl, ty) :: const_specs, rec_specs, dim_specs, _, tm :: tms ->
       let vty = V.eval tyenv ty in
       let varg = check_eval cx vty tm in
-      let cx' = Cx.def cx ~nm:(Some lbl) ~ty:vty ~el:varg in
+      let cx' = Cx.def cx' ~nm:(Some lbl) ~ty:vty ~el:varg in
```

Comment:
Is `cx'` being used at all in this definition (aside from just getting passed around)?